src/client/components/glossary/Glossary.jsx

```diff
@@ -8,6 +8,7 @@ const Glossary = () => {
   const definitions = definitionData.getDefinitionsArray();
   return (
     <div className='glossary'>
+      <div className='padList'/>
       { definitions.map(definition =>
```

Comment:
indent

Reply:
The `div` right above is self-closing, so I don't think so.  `{ definitions.map(definition =>` is the child of `<div className='glossary'>`.

Reply:
you're right, my fault